src/t_stream.c

```diff
@@ -2601,7 +2564,7 @@ void xgroupCommand(client *c) {
     streamCG *cg = NULL;
     char *opt = c->argv[1]->ptr; /* Subcommand name. */
     int mkstream = 0;
-    long long entries_read = 0;
+    long long entries_read = ULLONG_MAX;
```

Comment:
the var type is still signed. but maybe our decision to use unsigned was wrong.
maybe instead we should use LLONG_MIN as the special value?

Reply:
i think that it should be ok to always use `long long` and have `-1` denote that the value is invalid. LLONG_MAX is 9223372036854775807 which is way more than enough.
so we need to revert all related changes in networking.c

Reply:
maybe also `#define STREAM_CGROUP_INVALID_ENTRIES_READ -1`